torch/utils/dlpack.py

```diff
@@ -54,7 +54,7 @@ class DLDeviceType(enum.IntEnum):
 
 # TODO: add a typing.Protocol to be able to tell Mypy that only objects with
 # __dlpack__ and __dlpack_device__ methods are accepted.
-def from_dlpack(ext_tensor: Any) -> 'torch.Tensor':
+def from_dlpack(ext_tensor: Any, device: Any = None, copy: Optional[bool] = None) -> 'torch.Tensor':
```

Comment:
Docstring needs updating.
Also is the "device" type really any? We have  Device in our torch.types utils if needed.